AnkiDroid/src/main/java/com/ichi2/libanki/Note.java

```diff
@@ -151,17 +151,32 @@ public String joinedFields() {
     }
 
 
+    public int nbCards() {
+        return (int) mCol.getDb().queryLongScalar("SELECT count() FROM cards WHERE nid = ? ORDER BY ord", new Object[]{mId});
+    }
+
+    public ArrayList<Long> cids() {
```

Comment:
Optional: we might want to stay consistent with `cards()`

```suggestion
    public List<Long> cids() {
```

Reply:
Done